### ISSUE TITLE
Some songs in TH 12.3 is not accessible.

### ISSUE THREAD

**Issue by sclsj** [showermat/thplay#3]
<img width="1550" alt="TH 12 3" src="https://user-images.githubusercontent.com/38654636/81467641-a5f85680-920c-11ea-8381-2df1a1819d0e.png">

[dirs.txt](https://github.com/showermat/thplay/files/4602961/dirs.txt)

[ls -R.log](https://github.com/showermat/thplay/files/4602962/ls.-R.log)

Am I doing something wrong? The files come from a Touhou game collection and I had not changed them in any ways.

**Comment by sclsj** [comment]
[dirs.txt](https://github.com/showermat/thplay/files/4602961/dirs.txt)

**Comment by showermat** [comment]
Yes, those songs don't appear in the music archive of the game.  Maybe they're in one of the other archives, but I'm not an expert at reverse-engineering these things and the encryption they add makes it more difficult.  I never got to the bottom of where those songs are loaded from.  Maybe I can take a look at it again some day, and take another look at cracking the TH 135/145/155 encryption at the same time.
